package-lock.json

```diff
@@ -10756,7 +10756,7 @@
     },
     "meow": {
       "version": "3.7.0",
-      "resolved": "https://registry.npmjs.org/meow/-/meow-3.7.0.tgz",
+      "resolved": "http://registry.npmjs.org/meow/-/meow-3.7.0.tgz",
```

Comment:
I'm wondering why did we have so many http urls in this file. I some of our dependencies may have listed the wrong protocol in its dependencies file.

Reply:
It seems like a common NPM issue that these seem to switch back and forth, for whatever registry reason t_t

- https://npm.community/t/npm-install-downgrading-resolved-packages-from-https-to-http-registry-in-package-lock-json/1818
- https://discuss.circleci.com/t/npm-install-downgrades-package-lock-json-links-from-https-to-http/22508
- https://github.com/npm/npm/issues/20719